src/main/java/io/reactivex/internal/operators/maybe/MaybeFromAction.java

```diff
@@ -24,7 +26,7 @@
  *
  * @param <T> the value type
  */
-public final class MaybeFromAction<T> extends Maybe<T> {
+public final class MaybeFromAction<T> extends Maybe<T> implements Callable<T> {
```

Comment:
ScalarCallables are for assembly-time constants such as just(1). These Callables indicate they should be called when the subscription happens inside a `subscribeActual()`.